src/components/bs5/navbar/navbar.hbs

```diff
@@ -4,17 +4,21 @@
 <nav id="main-nav" class="navbar navbar-expand-lg" aria-label="Website navigation" role="navigation">
     <div class="container navbar-contain">
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
+
+            {{!-- Header for mobile navigation --}}
             <div class="navbar__header">
                 <h6 class="navbar__heading">Menu</h6>
                 <button aria-controls="main-nav" class="navbar__toggle navbar__toggle--close" data-bs-toggle="collapse"
                     data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
-                    aria-expanded="false" aria-label="Close navigation">
+                    aria-expanded="false" aria-label="Close menu">
```

Comment:
I think the main reason that the label was set to '**toggle** navigation' was to make it work for both expanded or unexpanded menu. You can change it to 'Toggle menu'. Otherwise we need some js code to change the label accordingly when the menu is open or close, to 'Close menu' and 'Open menu'.

Reply:
Added js to change the aria-label.